core/src/banking_stage.rs

```diff
@@ -458,7 +458,15 @@ impl BankingStage {
                     _ => (verified_receiver.clone(), ForwardOption::ForwardTransaction),
                 };
 
-                let mut packet_deserializer = PacketDeserializer::new(verified_receiver);
+                let runtime_config_tx_account_lock_limit = bank_forks
```

Comment:
if we have a runtime configuration, that overrides the feature-activated limit: `MAX_TX_ACCOUNT_LOCKS` or default of 64.

I want to either use the runtime configured value or `MAX_TX_ACCOUNT_LOCKS`. Otherwise I think we need a bank *during* deserialization to see if the feature is activated or not.

Reply:
Clarification on my thought process here:
* don't want to add bank_forks rwlock or cached root here - extra locking that I don't think is necessary
* if we use 64 and the feature is activated, then we'd filter out valid tx w/ 65-128 - i.e. will prevent actually activating the feature

This does mean that if we do *not* have a runtime config, and the feature is *not* activated, we will *not* filter out invalid tx with 65-128 locks. These will still be filtered out, prior to qos, by the changes in #28297 or #28303